docs/content/tutorial/manual-restart.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "Manually Restart PostgreSQL"
+date:
+draft: false
+weight: 105
+---
+
+There are times when you might need to manually restart PostgreSQL. This can be done utilizing PGO's rolling updates with minimal downtime by simply adding or updating a custom annotation for the cluster's instance sets `spec.metadata.annotations`, the Pod template will change for all instances and will trigger a rolling update for the cluster.
+
+For example, if I have a cluster named `hippo` in the namespace `postgres-operator`, I would set the following:
+
+```
+spec:
+  metadata:
+    annotations:
+      restart: "yes"
```

Comment:
:thinking: Thinking we'll want to use a timestamp (or similar) for the annotation here to indicate that a unique value (e.g. a unique timestamp) will need to be provided for each restart.

This also aligns with how we typically present the annotation value for triggering manual backups and restores.

Reply:
Agreed i'll get that updated